lib/term.css

```diff
@@ -1,17 +1,3 @@
-.terminal {
-  float: left;
-  border: #000000 solid 5px;
-  font-family: "DejaVu Sans Mono", "Liberation Mono", monospace;
-  font-size: 11px;
-  color: #f0f0f0;
-  background: #000000;
-}
-
-.terminal-cursor {
```

Comment:
Because cockpit.css has the similar rules which already overrode these [1]. Would you prefer to remove the ones there and keep them here in term.css?

[1] https://github.com/cockpit-project/cockpit/blob/master/src/base1/cockpit.css#L315